alert-common/src/main/java/com/synopsys/integration/alert/common/rest/model/FieldValueModel.java

```diff
@@ -60,7 +60,9 @@ public void setValues(Collection<String> values) {
     }
 
     public void setValue(String value) {
-        setValues(Set.of(value));
+        if (null != value) {
+            setValues(Set.of(value));
+        }
```

Comment:
Should this call `setValues` with `Set.of()` if the parameter is `null`?

Reply:
yes good catch.